Documentation/help_files/configure_orders_help.php

```diff
@@ -24,7 +24,7 @@
         @media only screen and (max-width: 768px) {
            [class*="col-"] {
            width: 100%;
-           text-align:left!important;
+           /*text-align:left!important;*/
```

Comment:
better to remove than comment out

Reply:
I will,  just left it till I was sure it did not affect the regular LTR views in responsive  mode